tests/ztest/busy_sim/boards/nrf52840dk_nrf52840.overlay

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+&timer0 {
+	status = "okay";
+	interrupts = <8 0>;
+};
+
+/ {
+	busy_sim: busy-sim {
```

Comment:
Just `busy-sim {` would be enough; the `busy_sim:` part is no longer required. I think it would be better to remove it just to avoid it getting copy/pasted everywhere unnecessarily by people who come later and want to use this.